ZAM_fnc_EZMLiteV2.1.1.sqf

```diff
@@ -6013,6 +6014,267 @@ MAZ_EZM_fnc_initFunction = {
 			},[]] call MAZ_EZM_fnc_createDialog;
 		};
 
+	comment "Cinematics";
+	comment "the plan here is as follows: when intro cinematic module is placed, a window pops up with one of two options: 'orbit' or 'dynamic'. in either case, the black bars appear for users, screen fades to black, and then the cinematic begins. The screen starts at black with a title and the name of the zeus'. If orbit is selected, the cinematic will be like a UAV flying above where the module was placed. Music choice can be selected, and up to 2 additional texts can be written to appear in the cinematic. once the cinematic ends, black screen appears again, and black bars fade.";
+	comment "maybe we should check if players are in a vehicle during cutscene, if so disable vehicle simulation";
```

Comment:
Yes. This would be necessary, and simple to implement. Ensure that enableSimulationGlobal is ran on the server.

All pilots would despise you for not disabling sim

Reply:
Just added it with
```sqf
{
	if !(_x == vehicle _x) then {
		[vehicle _x, false] remoteExec ["enableSimulationGlobal", 2];
	};
} forEach allPlayers;
```